react-forms/src/DatePicker.tsx

```diff
@@ -51,14 +52,13 @@ export default function DateTimePicker<T>(props: IProps<T>) {
     const recordFormat = props.Format !== undefined ? props.Format : "YYYY-MM-DD" + (props.Type === undefined || props.Type === 'date' ? "" : "[T]HH:mm:ss.SSS[Z]");
     const parse = (r: T) => moment(props.Record[props.Field] as any, recordFormat);
     const divRef = React.useRef<any | null>(null);
-    const recordChange = React.useRef<boolean>(false);
```

Comment:
I added this check in to combat this issue.
https://gridprotectionalliance.atlassian.net/issues/LSCVS-8?jql=project%20%3D%20LSCVS%20AND%20issuetype%20%21%3D%20%2210000%22%20AND%20%28fixVersion%20in%20unreleasedVersions%28%29%20OR%20fixVersion%20is%20EMPTY%29%20AND%20status%20%3D%20%2210005%22%20AND%20NOT%20%28updated%20%3E%3D%20-2w%20OR%20statusCategory%20%21%3D%20Done%29%20ORDER%20BY%20Rank%20ASC
Has this been tested to not crop back up due to the changes made?

Reply:
Go ahead and confirm, but I think with this rework that is no longer necessary

Reply:
It appears to be an issue in this commit

Reply:
Fix applied as per our conversation